### ISSUE TITLE
kucoin Invalid KC-API-TIMESTAMP

### ISSUE THREAD

**Issue by Elferboy** [ccxt/ccxt#23323]
### Operating System

10

### Programming Languages

PHP

### CCXT Version

4.3.73

### Description

Since today an error started to appear when requesting a balance from KuCoin exchange:
ccxt\InvalidNonce
kucoin Invalid KC-API-TIMESTAMP

The method `$exchange->load_time_difference();` does not help

### Code

```PHP
$exchangeClass = "\\ccxt\\kucoin";
$exchange      = new $exchangeClass([
  'apiKey'   => $account['api_key'],
  'secret'   => $account['api_secret'],
  'password' => $account['api_password'],
]);

$timeDifference = $exchange->load_time_difference();
// print_r($timeDifference); // returns: 5528

$balance = $exchange->fetchBalance([
  'type' => 'trade',
]);
print_r($balance);
// ccxt\InvalidNonce
// kucoin Invalid KC-API-TIMESTAMP
```

**Comment by carlosmiei** [comment]
Hello @Elferboy,
Nothing has changed on the CCXT side and I just tested and it worked.

Can you please check if your clock is synced with UTC? If not I would try to sync it before making the request.

```
 ph kucoin fetchBalance                                   
PHP v8.3.7
CCXT version :4.3.73
kucoin->fetchBalance()
Array
(
    [info] => Array
        (
            [code] => 200000
            [data] => Array
                (
                    [0] => Array
                        (
                            [id] => 64feeec4cd03f30007aa5a8f
                            [currency] => LTC
                            [type] => trade
                            [balance] => 0.211021
                            [available] => 0.211021
                            [holds] => 0
                        )

                    [1] => Array
                        (
                            [id] => 668635c2ff2dfe000796f545
                            [currency] => ADA
                            [type] => trade
                            [balance] => 20.47
                            [available] => 20.47
                            [holds] => 0
                        )

                    [2] => Array
                        (
                            [id] => 66
```

**Comment by Elferboy** [comment]
Thank you! I synchronized the time with the "time.windows.com" server and it started working fine. Too bad that the `load_time_difference()` method does not work for kucoin to equalize the difference.
Now the `load_time_difference()` method shows 1205 microseconds difference.
